frontend/testing/playwright/pages/OrgLibraryPage/CodeLists.ts

```diff
@@ -113,26 +117,28 @@ export class CodeLists extends BasePage {
     const toast = this.page.getByText(
       this.textMock('dashboard.org_library.code_list_upload_success'),
     );
-    await expect(toast).toBeHidden({ timeout: 8000 });
+    await expect(toast).toBeHidden({ timeout: TIMEOUT_FOR_TOAST_TO_DISAPPEAR });
   }
 
   public async clickOnCodeListAccordion(title: string): Promise<void> {
     await this.page
-      .getByTitle(
-        this.textMock('app_content_library.code_lists.code_list_accordion_title', {
+      .getByRole('button', {
+        name: this.textMock('app_content_library.code_lists.code_list_accordion_title', {
           codeListTitle: title,
         }),
-      )
+      })
       .click();
   }
 
-  public async verifyNumberOfRowsInTheCodelist(numberOfRows: number): Promise<void> {
-    for (let i = 1; i <= numberOfRows; i++) {
-      const valueRow = this.page.getByRole('textbox', {
-        name: this.textMock('code_list_editor.value_item', { number: i.toString() }),
-      });
+  public async verifyNumberOfRowsInTheCodelist(
+    numberOfRows: number,
+    codeListTitle: string,
+  ): Promise<void> {
+    const accordionTitle = this.page.getByRole('heading', { name: codeListTitle });
+    const accordion = accordionTitle.locator('xpath=..');
+    const table = accordion.locator('fieldset table');
+    const rows = table.locator('tbody tr');
```

Comment:
Har du prøvd med `getByRole` her?
```suggestion
    const table = accordion.getByRole('table');
    const rows = table.getByRole('row');
```